docs/src/UserGuide/write.md

```diff
@@ -28,6 +28,17 @@ savedataset(ds, path="ds.zarr", driver=:zarr)
 nothing # hide
 ````
 
+Save a dataset to Zarr format with compression:
+
+````@example write
+using Blosc
```

Comment:
you need to add `Blosc` to the Project toml, explicitly. Or, loaded it from whatever other package is also available.

Reply:
Adding Blosc to YAXArrays's Project.toml? or perhaps you mean YAXArrays/docs/Project.toml ?

Reply:
to docs 😄 .

Reply:
I think we can access the compressor within Zarr.jl directly. I will adapt the code